is_start_pred: true
diff_threshold: 0.8
max_rmse: 0.9
history_deque_len_cos: 250
history_deque_len_phase: 100
re_predict: false